src/components/CreateForm/CreateForm.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components/macro'
+import Button from '../Button/Button'
+import Input from '../Input/Input'
+import { cities } from '../../capstone.json'
+
+export default function CreateForm({ onCreateTrip }) {
+  return (
+    <Form data-testid="form" onSubmit={handleSubmit}>
+      <Heading>Plan your Trip</Heading>
+      <label>
+        City
+        <Select required name="city">
+          {cities.map(({ name, id }) => (
+            <option data-testid="select-option" key={id}>
+              {name}
+            </option>
+          ))}
+        </Select>
+      </label>
+      <label>
+        Start date
+        <Input required name="startDate" type="date" />
+      </label>
+      <label>
+        End date
+        <Input required name="endDate" type="date" />
+      </label>
+      <Button>Create Trip</Button>
+    </Form>
+  )
+  function handleSubmit(event) {
+    event.preventDefault()
+    const form = event.target
+    const { city, startDate, endDate } = form.elements
+
+    onCreateTrip({
+      city: city.value,
+      startDate: startDate.value,
+      endDate: endDate.value,
+    })
+    form.reset()
+    city.focus()
+  }
+}
+
+const Form = styled.form`
+  display: grid;
+  gap: 16px;
+  background: rgba(250, 250, 250, 0.9);
```

Comment:
langfristig brauchen wir noch Variables für die Farben. Hab ich auch noch nicht. Aber das wir dran denken, das nochmal in eine User Story zu packen. :)

Reply:
Kommt gleich mit auf die Liste meiner To Dos